l10n_it_ricevute_bancarie/models/riba_config.py

```diff
@@ -24,17 +24,19 @@ class RibaConfiguration(models.Model):
         "res.partner.bank",
         "Bank Account",
         required=True,
+        domain="[('company_id', '=', company_id)]",
```

Comment:
Qui e negli altri campi dove viene semplicemente aggiunto il dominio, penso sia meglio fare invece
```suggestion
        check_company=True,
```
come spiegato in https://github.com/OCA/l10n-italy/pull/3617#discussion_r1345398605.